dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -1472,4 +1476,53 @@ private static void addAnonymousPermissions(final Contentlet contentlet)
                 contentlet, APILocator.systemUser(), false);
     }
 
+    /**
+     * Method to test: {@link HTMLPageAssetRenderedAPI#getPageHtml(PageContext, HttpServletRequest, HttpServletResponse)}
+     * When: TimeMachine in running and request a existing Page
+     * Should: Return the page
+     */
+    @Test
+    public void testingTimeMachine () throws Exception{
+
+        final Container container = createContainer();
+        final Template template = createTemplate(container);
+
+        Config.setProperty(contentFallbackProperty, false);
+        Config.setProperty(pageFallbackProperty, false);
+
+        final String pageName = "test1Page-"+System.currentTimeMillis();
+        final HTMLPageAsset pageEnglishVersion = createHtmlPageAsset(template, pageName, 1);
+
+        createMultiTree(pageEnglishVersion.getIdentifier(), container.getIdentifier());
+
+        final HttpServletRequest mockRequest = new MockSessionRequest(
+                    new MockAttributeRequest(new MockHttpRequest("localhost", "/").request()).request()
+                )
+                .request();
+
+
+        Mockito.when(mockRequest.getParameter("host_id")).thenReturn(site.getIdentifier());
+        mockRequest.setAttribute(WebKeys.HTMLPAGE_LANGUAGE, "1");
+        HttpServletRequestThreadLocal.INSTANCE.setRequest(mockRequest);
+        final HttpServletResponse mockResponse = mock(HttpServletResponse.class);
+
+        Calendar tomorrow = Calendar.getInstance();
+        tomorrow.add(Calendar.DATE, 1);
+
+        //mockRequest.getSession().setAttribute("tm_date", Long.toString(tomorrow.getTime().getTime()));
+        //mockRequest.getSession().setAttribute("tm_lang", "1");
+
+        FilterChain chain = Mockito.mock(FilterChain.class);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Mockito.mock' due to existing static import 'org.mockito.Mockito.mock'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5474742)